### PR TITLE
add fees in the right way to all file services

### PR DESCRIPTION
**Pull request by povolev15**
Implemented the fees as it supposed to be in the services

**Related issue(s)**:

Fixes #8424

### CODE REVIEW

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/query/QueryContextImpl.java [hunk 1]
```diff
@@ -130,4 +132,10 @@ public ExchangeRateInfo exchangeRateInfo() {
         }
         return exchangeRateInfo;
     }
+
+    @NonNull
+    @Override
+    public FeeCalculator feeCalculator(@NonNull SubType subType) {
+        return null;
```
Comment (jsync-swirlds) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/query/QueryContextImpl.java:139]: Is there is an issue in place to fill this in?

Comment (povolev15) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/query/QueryContextImpl.java:139]: I think Michal or Richard will it in. I can definitely can create one

**Review by iwsimon — DISMISSED** [review]
LGTM

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/query/QueryContextImpl.java (hunk 1 above)
Comment (povolev15) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/query/QueryContextImpl.java:139]: ticket created https://github.com/hashgraph/hedera-services/issues/8543

**Review by iwsimon — APPROVED** [review]

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java [hunk 2]
```diff
@@ -146,40 +146,43 @@ public void handle(@NonNull final HandleContext handleContext) throws HandleExce
...
+                : 0;
 
         /**
          * TODO: revisit after modularizaion completed
          * PR conversation: 8089
          */
         final long effectiveLifeTime;
+        // TODO revert it back to config when it will be exist in FeeContextImpl - 8545 issue
+        //        final var fileServiceConfig = feeContext.configuration().getConfigData(FilesConfig.class);
+        final var file = feeContext
```
Comment (netopyr) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java:164]: Is it really necessary to read the whole file here? You just need `fileNum`, right? Can we do `op.fileAppendOrThrow().fileIDOrThrow().fileNum()` instead?

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java [hunk 3]
```diff
@@ -71,6 +78,19 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
         }
     }
 
+    public Fees computeFees(@NonNull QueryContext queryContext) {
+        final var query = queryContext.query();
+        final var fileStore = queryContext.createStore(ReadableFileStore.class);
+        final var op = query.fileGetContentsOrThrow();
+        final var fileId = op.fileIDOrThrow();
+        final var responseType = op.headerOrElse(QueryHeader.DEFAULT).responseType();
+        final FileContents fileContents = contentFile(fileId, fileStore);
```
Comment (netopyr) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java:87]: We need to consider our design. If we have to do the work twice (reading the file content) just to calculate the fees, something is wrong. For now, it is ok, but we need to revisit this later.

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java [hunk 4]
```diff
@@ -81,6 +88,18 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
         }
     }
 
+    public Fees computeFees(@NonNull QueryContext queryContext) {
+        final var query = queryContext.query();
+        final var fileStore = queryContext.createStore(ReadableFileStore.class);
+        final var op = query.fileGetInfoOrThrow();
+        final var fileId = op.fileIDOrThrow();
+        final File file = fileStore.getFileLeaf(fileId);
```
Comment (netopyr) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java:96]: It really hurts to see this. 🙂 Let's discuss this in one of our calls.

Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java:96]: I agree with you 100%

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java (hunk 2 above)
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java:164]: but than I need 5 lines below file.expirationSecond()

**Review by jsync-swirlds — APPROVED** [review]

**Review by vtronkov — APPROVED** [review]

**Review by jsync-swirlds — APPROVED** [review]

**Review by iwsimon — APPROVED** [review]
LGTM

**Pull request merged**